### PR TITLE
feat(audio): batch 2 of native updates

### PR DESCRIPTION
**Pull request by AvarianKnight**
Before you submit this PR, please make sure:

- You have read the contribution guidelines
- You include an example that validates your change
- Your English is grammatically correct

### CODE REVIEW

File: AUDIO/DisablePedPainAudio.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ ns: AUDIO
 
 ```c
 // 0xA9A41C1E940FB0E8 0x3B8E2D5F
-void DISABLE_PED_PAIN_AUDIO(Ped ped, BOOL toggle);
+void DISABLE_PED_PAIN_AUDIO(Ped ped, BOOL shouldDisable);
 ```
```
Comment (PsychoShock) [inline AUDIO/DisablePedPainAudio.md:10]: Description can be added - `Disable ped pain audio`

Comment (PsychoShock) [inline AUDIO/DisablePedPainAudio.md:10]: Description can be added - `Disable ped pain audio`

Comment (AvarianKnight) [reply AUDIO/DisablePedPainAudio.md:10]: The description would be redundant.

**Comment by 4mmonium** [comment]
LGTM, merging!

**Pull request merged**